presto-iceberg/src/test/java/com/facebook/presto/iceberg/nessie/TestNessieMultiBranching.java

```diff
@@ -157,26 +157,13 @@ public void testTableDataVisibility()
         // retrieve the second to the last commit hash and query the table with that hash
         List<LogResponse.LogEntry> logEntries = nessieApiV1.getCommitLog().refName(two.getName()).get().getLogEntries();
         assertThat(logEntries).isNotEmpty();
-        String hash = logEntries.get(1).getCommitMeta().getHash();
```

Comment:
@agrawalreetika are you planning to revert these changes in this PR?

Reply:
Yes, @nastra I have taken out this commit from this PR.